tests/integration/components/high-charts-test.js

```diff
@@ -0,0 +1,16 @@
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, skip } from 'qunit';
+
+module('Integration | Component | high-charts', function (hooks) {
+  setupRenderingTest(hooks);
+
+  skip('it renders', async function (assert) {
+    await render(hbs`
+      <HighCharts />
```

Comment:
Rendering the component in a test currently results in the error `calling set on destroyed object`. This is a known issue that, I think, can be better fixed by updating the component to Glimmer.